Cargo.toml

```diff
@@ -26,9 +26,9 @@ anyhow = "1"
 
 serde_json = "1"
 
-skar-client = "0.5.2"
-skar-net-types = "0.1.1"
-skar-format = "0.2.0"
+skar-client = "0.6"
```

Comment:
I'm always used to seeing the `.0` at the end, does it not make a difference?

Reply:
It is always "^x.x.x" implicitly so if we have a minor version the patch version doesn't matter

Reply:
I removed them because they get confusing